avm/res/signal-r-service/signal-r/main.bicep

```diff
@@ -346,39 +375,7 @@ output privateEndpoints array = [
 //   Definitions   //
 // =============== //
 
-type lockType = {
-  @description('Optional. Specify the name of lock.')
-  name: string?
-
-  @description('Optional. Specify the type of lock.')
-  kind: ('CanNotDelete' | 'ReadOnly' | 'None')?
-}?
-
-type roleAssignmentType = {
-  @description('Optional. The name (as GUID) of the role assignment. If not provided, a GUID will be generated.')
-  name: string?
-
-  @description('Required. The role to assign. You can provide either the display name of the role definition, the role definition GUID, or its fully qualified ID in the following format: \'/providers/Microsoft.Authorization/roleDefinitions/c2f4ef07-c644-48eb-af81-4b1b4947fb11\'.')
-  roleDefinitionIdOrName: string
-
-  @description('Required. The principal ID of the principal (user/group/identity) to assign the role to.')
-  principalId: string
-
-  @description('Optional. The principal type of the assigned principal ID.')
-  principalType: ('ServicePrincipal' | 'Group' | 'User' | 'ForeignGroup' | 'Device')?
-
-  @description('Optional. The description of the role assignment.')
-  description: string?
-
-  @description('Optional. The conditions on the role assignment. This limits the resources it can be assigned to. e.g.: @Resource[Microsoft.Storage/storageAccounts/blobServices/containers:ContainerName] StringEqualsIgnoreCase "foo_storage_container".')
-  condition: string?
-
-  @description('Optional. Version of the condition.')
-  conditionVersion: '2.0'?
-
-  @description('Optional. The Resource Id of the delegated managed identity resource.')
-  delegatedManagedIdentityResourceId: string?
-}[]?
+import { lockType, roleAssignmentType, managedIdentityAllType } from 'br/public:avm/utl/types/avm-common-types:0.1.0'
```

Comment:
Let's agree on this, it would be great to follow one method

Reply:
Except for the new version comment and this discussion, it looks good to me. @ReneHezser @AlexanderSehr how are we unblocking the above discussion?

Reply:
I could've sworn I commented on this somewhere else already.
@ReneHezser, I think we agreed in splitting the imports, right? I know that I already tried and failed to find the comment where we discussed this last time - but I can try again 😄

Reply:
@AlexanderSehr I don't recall the decision but changed to an import per line and updated the version.

Reply:
If willing to go with splitting imports, I'd suggest to keep it consistent with other modules where this was already adopted and place the import right before its corresponding parameter description. For example like [here](https://github.com/Azure/bicep-registry-modules/blob/main/avm/res/automation/automation-account/main.bicep#L60)

Reply:
I've moved them up, so they are right where they are needed.